### PR TITLE
client: Remove PAT option for user added code

### PR DESCRIPTION
**Pull request by ryanslade**
Removed the option to add via a personal access token (PAT). This allows
us to:

1. Removed the code host connection buttons
2. Switch from a dropdown to a single connect or remove button

Closes: https://github.com/sourcegraph/sourcegraph/issues/21327

Before:

Screenshot 2021-05-25 at 11 48 32

After:

Screenshot 2021-05-25 at 11 38 12

### CODE REVIEW

File: client/web/src/user/settings/codeHosts/UserAddCodeHostsPage.tsx [hunk 1]
```diff
@@ -186,39 +185,6 @@ export const UserAddCodeHostsPage: React.FunctionComponent<UserAddCodeHostsPageP
         [authProvidersByKind]
     )
 
-    const codeHostOAuthButtons = isServicesByKind(statusOrError)
```
Comment (ryanslade) [inline client/web/src/user/settings/codeHosts/UserAddCodeHostsPage.tsx:189]: @artemruts I removed this as it was flagged as dead code.

But, should we not still keep the event logger code? I'm not sure why we weren't doing this when code hosts were added via the dropdown before.

**Comment by ryanslade** [comment]
@quinnkeast Anything I need to change in the design?

**Comment by quinnkeast** [comment]
Looks good! Do we have a reversed spinner that could be used on the primary button here?

**Comment by ryanslade** [comment]
> Looks good! Do we have a reversed spinner that could be used on the primary button here?

I'm not sure what you mean by a reversed spinner? When you click "Connect" now a circular spinner appears on the right side of the button:

**Comment by quinnkeast** [comment]
Sorry, should've explained more clearly. Right now, the spinner's a grey colour, which has poor contrast with the background of the button. Reversed here means a white-based spinner so that there's better contrast.

File: client/web/src/user/settings/codeHosts/UserAddCodeHostsPage.tsx (hunk 1 above)
Comment (artemruts) [reply client/web/src/user/settings/codeHosts/UserAddCodeHostsPage.tsx:189]: Ryan, I think we simply missed it or there's a chance we ask by analytics to place it there.
Let's add it to the other place too that way tracking is constant. 

EDIT:

I was pretty sure I added this line until I ran `git log -S` 😄 &nbsp;and found [this PR](https://github.com/sourcegraph/sourcegraph/pull/20003/files#diff-770d0e37465272d5c781cd854a29bfb47042e39022339d8f88a7151ac980c440R198).

**Comment by artemruts** [comment]
> Sorry, should've explained more clearly. Right now, the spinner's a grey colour, which has poor contrast with the background of the button. Reversed here means a white-based spinner so that there's better contrast.



> Sorry, should've explained more clearly. Right now, the spinner's a grey colour, which has poor contrast with the background of the button. Reversed here means a white-based spinner so that there's better contrast.

@ryanslade I think we can add `theme-dark` CSS class to the `LoadingSpinner` component - it'll make spinner much lighter. It looks good with both dark and light themes because of the blueish `Connect` buttons.

**Review by artemruts — APPROVED** [review]

File: client/web/src/user/settings/codeHosts/UserAddCodeHostsPage.tsx (hunk 1 above)
Comment (ryanslade) [reply client/web/src/user/settings/codeHosts/UserAddCodeHostsPage.tsx:189]: Hah, didn't expect that :)

**Pull request merged**